include/bluetooth/mesh/model_types.h

```diff
@@ -22,7 +22,20 @@ extern "C" {
 /** Maximum permissible transition time in milliseconds */
 #define BT_MESH_MODEL_TRANSITION_TIME_MAX_MS (10 * 60 * MSEC_PER_SEC * 0x3e)
 
-/** Generic Transition parameters for the model messages. */
+/** Delay field step factor in milliseconds */
+#define BT_MESH_MODEL_DELAY_TIME_STEP_FACTOR_MS (5)
+/** Maximum encoded value of the delay field */
+#define BT_MESH_MODEL_DELAY_TIME_STEP_MAX (0xFF)
+/** Maximum permissible delay time in milliseconds */
+#define BT_MESH_MODEL_DELAY_TIME_MAX_MS                                        \
+	(BT_MESH_MODEL_DELAY_TIME_STEP_MAX *                                   \
+	 BT_MESH_MODEL_DELAY_TIME_STEP_FACTOR_MS)
```

Comment:
Does this really need all three #defines? The first two are only used for computing the third as far as I can tell, I see the TRANSITION_TIME_MAX_MS is just computed as is. Could optionally clarify the factors with comments while inlining the constants.

Reply:
It is only that the spec specifies that the delay should be in steps of 5 ms, so get an understanding of the max value it is easier to specify it with the same values as from the spec. It consist of a uint8_t in 5 ms steps. It does not increase code size and it makes it easier to understand how the value is calculated.

Reply:
Could also use `UINT8_MAX` instead of `0xFF`, but I'm okay with the current version.